CIFAR10/inference/Baidu_USA_GAIT_LEOPARD_team.txt

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
The file extension should be `.json` not `.txt`. None of the unittests picked up this file because of that. Please fix and make sure the unit tests pass. From the root directory of this repo, you can run the unittests locally with the following:

```
pip install -r requirements.txt
pytest
```